frontend/src/components/AlertBadge.tsx

```diff
@@ -0,0 +1,34 @@
+import { Row, Alert } from "react-bootstrap";
+
+interface Props {
+  text: string;
+  variant: string;
+  active: boolean;
+}
+
```

Comment:
Alert doesnt work when user inputs new data or removes the alert by clicking the x button and resubmitting data.